extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -103,6 +106,12 @@
 
     static final DotName OBJECT_MAPPER = DotName.createSimple("com.fasterxml.jackson.databind.ObjectMapper");
 
+    // TODO do we want this?
```

Comment:
Yes we do and we can remove this comment :).

Reply:
Thanks! :-)